shared/chat/conversation/index.desktop.js

```diff
@@ -175,7 +176,11 @@ export default branch(
   renderComponent(NoConversation),
   branch(
     (props: Props) => !!props.rekeyInfo,
-    renderComponent(ParticipantRekey),
+    branch(
```

Comment:
For general discussion: I think this is getting really confusing with the nesting. Took me way too long to figure out what this branching was doing and what  `withFocusHandlers` applied to. What is the advantage of expressing the logic in this way vs. putting it in the caller?